addons/mail/static/src/models/message.js

```diff
@@ -529,6 +530,27 @@ registerModel({
             }
             return false;
         },
+        /**
+         * @private
+         * @returns {MessageTrackingValue}
+         */
+        _computeLastTrackingValue() {
+            const {
+                length: l,
+                [l - 1]: lastTrackingValue,
+            } = this.orderedTrackingValues;
+            if (lastTrackingValue) {
+                return replace(lastTrackingValue);
+            }
+            return clear();
+        },
+        /**
+         * @private 
+         * @returns {FieldCommand}
+         */
+        _computeOrderedTrackingValues() {
+            return replace(this.trackingValues);
+        },
```

Comment:
useless compute/field, move the `sort` to `trackingValues`?
(and rename it if we really want to clarify it, but I wouldn't mind if we don't rename)